internal/pkg/cli/svc_init_test.go

```diff
@@ -474,7 +474,7 @@ func TestSvcInitOpts_Ask(t *testing.T) {
 			inDockerfilePath: "",
 
 			setupMocks: func(m initSvcMocks) {
-				m.mockMftReader.EXPECT().ReadWorkloadManifest(wantedSvcName).Return(nil, &workspace.ErrFileNotExists{FileName: wantedSvcName})
+				m.mockMftReader.EXPECT().ReadWorkloadManifest(wantedSvcName).Return(nil, &workspace.ErrWorkspaceNotFound{})
```

Comment:
maybe we should keep one test case for ErrFileNotExists and update only one file for ErrWorkspaceNotFound?

Reply:
I updated the code, so that `svc_init_test` (here) return the error `ErrFileNotExists`, and `job_init_test` returns `ErrWorkspaceNotFound` - let me know if this is not what you meant!